.happy/terraform/modules/sfn_config/nextstrain-dbg.wdl

```diff
@@ -67,14 +67,17 @@ task nextstrain_workflow {
     )
 
     # set up ncov. keep the fetch command in case want to overwrite the version in Docker image
-    mkdir -p /ncov/my_profiles/aspen /ncov/results
-#    (cd /ncov &&
-#     git init &&
-#     git fetch --depth 1 https://github.com/nextstrain/ncov.git df90b457f48ef3d7500927656536cacb16c9a83f &&
-#     git checkout FETCH_HEAD
-#    )
+    (rm -r /ncov
+     mkdir /ncov
```

Comment:
remove the original folder otherwise sometimes get error complaining about merge conflict.....